nucliadb/nucliadb/search/search/query.py

```diff
@@ -320,3 +321,24 @@ async def get_default_min_score(kbid: str) -> float:
         # B/w compatible code until we figure out how to
         # set default min score for old on-prem kbs
         return fallback
+
+
+def record_features_counter_metric(pb_query: SearchRequest, counter: Counter):
+    if len(pb_query.filter.tags):
+        counter.inc({"type": "filters"})
+        if any(tag.startswith("/l/") for tag in pb_query.filter.tags):
+            counter.inc({"type": "labels_filters"})
+        if any(tag.startswith("/e/") for tag in pb_query.filter.tags):
+            counter.inc({"type": "entities_filters"})
+    if len(pb_query.key_filters):
```

Comment:
small suggestion, maybe the block below can be simplified for readability

```
for field in ("key_filters", "vectorset", ..):
    if getattr(pb_query, field, False):
        counter.inc({"type": field})
```

Reply:
mh, actually, if you don't have any big objection with it, with metrics recording code I prefer being more verbose and explicit

Reply:
nevermind, I changed the implementation to do the increments in the pb_query function